src/utils/Processing.ts

```diff
@@ -220,16 +228,41 @@ export function loadAutomaticDependencies(
   defs: FHIRDefinitions
 ): Promise<void | FHIRDefinitions>[] {
   return AUTOMATIC_DEPENDENCIES.map(dep => {
-    if (configuredDependencies.some(cd => cd.packageId === dep.packageId)) {
+    const alreadyConfigured = configuredDependencies.some(cd => {
+      // hl7.some.package, hl7.some.package.r4, and hl7.somepackage.r5 all represent the same content,
+      // so they are essentially interchangable and we should allow for any of them in the config.
```

Comment:
If the above question warrants a change, how do you feel about this monumental change while you're already in the file?
```suggestion
      // so they are essentially interchangeable and we should allow for any of them in the config.
```